client/state/rewind/retention/test/reducer.js

```diff
@@ -0,0 +1,40 @@
+import {
+	JETPACK_BACKUP_RETENTION_UPDATE,
+	JETPACK_BACKUP_RETENTION_UPDATE_ERROR,
+	JETPACK_BACKUP_RETENTION_UPDATE_SUCCESS,
+} from 'calypso/state/action-types';
+import retentionReducer from '../reducer';
+
+describe( 'rewind.retention reducers', () => {
+	describe( 'lastBackupSize', () => {
```

Comment:
```suggestion
	describe( 'updateBackupRetentionRequestStatus', () => {
```

Shouldn't we use `updateBackupRetentionRequestStatus` here?

Reply:
✅